docs/website/docs/reference/frequently-asked-questions.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Frequently Asked Questions
+description: Questions asked frequently by users in technical help or github issues
+keywords: [faq, usage information, technical help]
+---
+
+
+#### Q: Can I configure different nesting levels for each resource in a source within DLT?
```

Comment:
1. "Q" is redundant: there's no need to indicate that this is a question since the sentence ends with a question mark.  
2. Use H2 level for the headings. The TOC will be autogenerated in the right sidebar which helps with navigation on this page.
```suggestion
## Can I configure different nesting levels for each resource in a source within DLT?
```

Reply:
Done but I hope it does not look too big.